eva/parser/parser_visitor/_create_statements.py

```diff
@@ -94,11 +94,19 @@ def visitColumnDefinition(self, ctx: evaql_parser.ColumnDefinitionContext):
 
         column_constraint_information = ColConstraintInfo()
 
+        not_null_set = False
         for i in range(constraint_count):
             return_type = self.visit(ctx.columnConstraint(i))
             if return_type == ColumnConstraintEnum.UNIQUE:
-
                 column_constraint_information.unique = True
+                column_constraint_information.nullable = False
+                not_null_set = True
+            elif return_type == ColumnConstraintEnum.NULLNOTNULL:
```

Comment:
Can we call it `NOTNULL`?

Reply:
Yeah I was thinking the same why it was named this way originally.